evap/results/templates/results_semester_detail.html

```diff
@@ -47,8 +51,22 @@
                                 <td class="text-center"><div class="grade-bg" style="background-color: {{ course.avg_grade|gradecolor }};">{{ course.avg_grade|floatformat:1 }}</div></td>
                                 <td class="text-center"><div class="deviation-bg" style="background-color: {{ course.avg_deviation|deviationcolor }};">{{ course.avg_deviation|floatformat:1 }}</div></td>
                             {% else %}
-                                <td class="text-center"><div class="grade-bg grade-bg-disabled" data-toggle="tooltip" data-placement="left" title="{% trans "Not enough answers were given." %}">&mdash;</div></td>
-                                <td class="text-center"><div class="deviation-bg deviation-bg-disabled" data-toggle="tooltip" data-placement="left" title="{% trans "Not enough answers were given." %}">&mdash;</div></td>
+                                <td class="text-center"><div class="grade-bg grade-bg-disabled" data-toggle="tooltip" data-placement="left" title="
+                                    {% if course.is_private %}
+                                        {% trans "The results of this course are private." %}
+                                    {% else %}
+                                        {% trans "Not enough answers were given to publish the results." %}
+                                    {% endif %}">
+                                    &mdash;
+                                </div></td>
+                                <td class="text-center"><div class="deviation-bg deviation-bg-disabled" data-toggle="tooltip" data-placement="left" title="
+                                    {% if course.is_private %}
+                                        {% trans "The results of this course are private." %}
+                                    {% else %}
+                                        {% trans "Not enough answers were given to publish the results." %}
+                                    {% endif %}">
```

Comment:
any way to deduplicate this if else?

besides, why is the course shown at all here if it is private?